File: csharp/aspnet-core-example/Models/TTSJobRequest.cs
using System.Text.Json.Serialization;

namespace CreoleCentricExample.Models;

public class TTSJobRequest
{
    [JsonPropertyName("text")]
    public string Text { get; set; } = string.Empty;

    [JsonPropertyName("voice_id")]
    public string VoiceId { get; set; } = "i4mRPwKM2yHwXhbmkN514"; // Xavier Bruneau default

    [JsonPropertyName("model_id")]
    public string ModelId { get; set; } = "ccl_ht_v100"; // Default Haitian Creole model

    [JsonPropertyName("speed")]
    public double Speed { get; set; } = 1.0;

    [JsonPropertyName("pitch")]
    public double Pitch { get; set; } = 1.0;
}

File: java/spring-boot-example/src/main/java/com/creolecentric/example/model/TTSJobRequest.java
package com.creolecentric.example.model;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

@Data
public class TTSJobRequest {
    private String text;

    @JsonProperty("voice_id")
    private String voiceId = "i4mRPwKM2yHwXhbmkN514"; // Xavier Bruneau default

    @JsonProperty("model_id")
    private String modelId = "ccl_ht_v100"; // Default Haitian Creole model

    private Double speed = 1.0;
    private Double pitch = 1.0;
}
